lib/browser/lighthouse.js

```diff
@@ -32,114 +32,138 @@
 
 'use strict';
 
-const assert = require('assertive');
+const assert = require('assert');
 const isEmpty = require('lodash/isEmpty');
+const flatMap = require('lodash/flatMap');
 
+/**
+ * @typedef import('lighthouse/types/lhr') LH
+ */
+
+const defaultConfig = {
+  extends: 'lighthouse:default',
+  settings: {
+    onlyCategories: [],
+    throttlingMethod: 'simulate',
+  },
+};
+
+const defaultFlags = {
+  chromeFlags: [
+    '--disable-gpu',
+    '--headless',
+    '--disable-storage-reset',
+    '--enable-logging',
+    '--disable-device-emulation',
+    '--no-sandbox',
+  ],
+};
+
+/**
+ * @param {string} type
+ * @param {Array<string>} [skipAudits]
+ * @return {Object}
+ */
+function getConfigByType(type, skipAudits) {
+  const config = { ...defaultConfig, settings: defaultConfig.settings };
```

Comment:
This is a no-op.  Given below where you're pushing onto a deep array, you should be deep-cloning this.  Either use `lodash.clonedeep`, or JSON stringify/parse it.